PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -368,6 +367,7 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
       float ecalEnergy = 0;
       float hcalEnergy = 0;
       float sumEtaTimesEnergy = 0.;
+      float sumPhiTimesEnergy = 0.;
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.